src/homework-1/task3/task-3-2.js

```diff
@@ -0,0 +1,21 @@
+import fs from 'fs'
+import csv from 'csvtojson'
+import { pipeline } from 'stream'
+
+const sourceFile = './resources/books.csv'
+const targetFile = './resources/books.txt'
+
+pipeline(
+  fs.createReadStream(sourceFile),
+  csv(),
+  fs.createWriteStream(targetFile),
+  (err) => processFinishedPipeline(err)
```

Comment:
No need to write `(err) => processFinishedPipeline(err)`, `processFinishedPipeline` is enough

Reply:
Fixed